src/authorization/exception/user.exception.ts

```diff
@@ -1,7 +1,14 @@
+import { BadRequestException } from '@nestjs/common';
 import { NotFoundException } from '@nestjs/common';
 
 export class UserNotFoundException extends NotFoundException {
   constructor(userId: string) {
     super(`User ${userId} not found`);
   }
 }
+
+export class AccountIsInactive extends BadRequestException {
+  constructor() {
+    super(`Account is inactive`);
```

Comment:
What would be the status code thrown for this exception? `BadRequestException` throws `400` which indicates that the request is incorrect. Can we put in a more appropriate status code here?